_analyzers/token-filters/kuromoji-completion.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+title: Kuromoji completion
+parent: Token filters
+nav_order: 230
+---
+
+# Kuromoji completion token filter
+
+The `kuromoji_completion` token filter is used to stem Katakana words in Japanese, which are often used to represent foreign words or loanwords. This filter is especially powerful for autocompletion or suggest queries, where partial matches on Katakana words can be expanded to include full forms.
+
```

Comment:
Line 10, last sentence: "in which partial Katakana word matches can be expanded to their full form"?

Reply:
I think the original is easier to parse.

Reply:
@kolchfa-aws Ok, but we still need "their" before "full".